src/t_cose_crypto.h

```diff
@@ -563,6 +580,26 @@ t_cose_algorithm_is_ecdsa(int32_t cose_algorithm_id)
     return t_cose_check_list(cose_algorithm_id, ecdsa_list);
 }
 
+static inline bool
+t_cose_algorithm_is_rsassa_pss(int32_t cose_algorithm_id)
+{
+    /* The simple list of COSE alg IDs that use RSASSA-PSS */
+    static const int32_t rsa_list[] = {
+#ifndef T_COSE_DISABLE_PS256
+        COSE_ALGORITHM_PS256,
+#endif
+#ifndef T_COSE_DISABLE_PS384
+        COSE_ALGORITHM_PS384,
+#endif
+#ifndef T_COSE_DISABLE_PS512
+        COSE_ALGORITHM_PS512,
+#endif
+        0}; /* 0 is a reserved COSE alg ID ans will never be used */
```

Comment:
```suggestion
        0}; /* 0 is a reserved COSE alg ID and will never be used */
```

Reply:
Done. Also fixed in `t_cose_algorithm_is_ecdsa` where I'd copied the typo from.